src/component.js

```diff
@@ -65,28 +68,25 @@ export default class Component {
     }
 
     getUnobservedData() {
-        let rawData = {}
+        let unwrapedData = this.membrane.unwrapProxy(this.$data)
```

Comment:
typo across this function
```suggestion
        let unwrappedData = this.membrane.unwrapProxy(this.$data)
```

Reply:
Good catch, thanks!